From 1bd61c0f4f52031f8d94cb20eff296d0b833957e Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Tue, 25 Jun 2024 13:10:53 -0700
Subject: [PATCH] Docs/mm/damon/design: Remove 'Programmable Modules' section
 in favor of 'Modules' section

'Programmable Modules' section provides high level description of the
DAMON API-based kernel modules layer.  But 'Modules' section that
dedicated for the layer and describe everything about the layer
including that explained on 'Programmable Modules' section follows is at
the end of the document.  The above brief summary of the layers has a
link to the 'Modules' section, so browsing to the section is not that
difficult.  Remove 'Programmable Modules' section in favor of 'Modules'
section and reducing duplicates.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/mm/damon/design.rst | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/Documentation/mm/damon/design.rst b/Documentation/mm/damon/design.rst
index f7029bc840ce..39e686c6862d 100644
--- a/Documentation/mm/damon/design.rst
+++ b/Documentation/mm/damon/design.rst
@@ -33,16 +33,6 @@ DAMON subsystem is configured with three layers including
   layer.
 
 
-Programmable Modules
---------------------
-
-Core layer of DAMON is implemented as a framework, and exposes its application
-programming interface to all kernel space components such as subsystems and
-modules.  For common use cases of DAMON, DAMON subsystem provides kernel
-modules that built on top of the core layer using the API, which can be easily
-used by the user space end users.
-
-
 .. _damon_operations_set:
 
 Operations Set Layer
-- 
2.39.2

